qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -252,12 +252,12 @@ def sparse_operation(state: Tensor, values: dict[str, Tensor] = None) -> Tensor:
                 convert_block(block, n_qubits, config),
             )
 
-    def forward(self, state: Tensor, values: dict[str, Tensor]) -> Tensor:
-        return pyq.overlap(state, self.operation(state, values))
-
     def run(self, state: Tensor, values: dict[str, Tensor]) -> Tensor:
         return self.operation(state, values)
 
+    def forward(self, state: Tensor, values: dict[str, Tensor]) -> Tensor:
```

Comment:
Not sure why you changed this in this PR but fine for me?

Reply:
haha you know while youre cleaning the kitchen you end up cleaning the bathroom too...